### DISCUSSION TITLE
Questions about PDS tools functions

### DISCUSSION THREAD

**Discussion by jonathangebru** [pegasystems/pega-datascientist-tools#147] [Q&A]
CDHSample.plotPredictorPerformanceHeatmap() => What exactly is does performance mean for these predictions. Is this the contribution to the propensity. I can't find how this is calculated. 

 CDHSample.plotScoreDistribution => I don't understand the range that is described within these graphs.

**Comment by operdeck** [comment]
Hi, for predictor performance we use the univariate performance, just like you see it in Prediction Studio, this is not directly related to the propensity contribution, instead, the performance (ROC AUC) is a measure of discrimination.

The score distribution plot is similar to the one you see in Prediction Studio. The x-axis represents the average log odds contribution of all the active predictors as also explained in: https://pegasystems.github.io/pega-datascientist-tools/Python/articles/ADMExplained.html, but if you haven't looked at it, check out the Data Science track on Pega Academy first.

Hope this helps,

-Otto
